core/src/main/scala/com/spotify/featran/transformers/HashNHotEncoder.scala

```diff
@@ -24,20 +24,43 @@ import scala.collection.SortedSet
 
 object HashNHotEncoder {
   /**
-   * Transform a collection of categorical features to binary columns, with at most N one-values.
-   * Similar to [[NHotEncoder]] but uses MurmursHash3 to hash features into buckets to reduce CPU
-   * and memory overhead.
-   *
-   * Missing values are transformed to [0.0, 0.0, ...].
-   *
-   * @param hashBucketSize number of buckets, or 0 to infer from data with HyperLogLog
-   */
-  def apply(name: String, hashBucketSize: Int = 0): Transformer[Seq[String], HLL, Int] =
-    new HashNHotEncoder(name, hashBucketSize)
+    * Transform a collection of categorical features to binary columns, with at most N one-values.
+    * Similar to [[NHotEncoder]] but uses MurmursHash3 to hash features into buckets to reduce CPU
+    * and memory overhead.
+    *
+    * Missing values are transformed to [0.0, 0.0, ...].
+    *
+    * If hashBucketSize is inferred with HLL, the estimate is scaled by sizeScalingFactor
+    * to reduce the number of collisions.
+    *
+    * Rough table of relationship of scaling factor to % collisions:
```

Comment:
Add that it's "measured from a corpus of 466544 English words".
Also the numbers are based on size from 1<<19 to 1<<30, not exact multiples of 466544. Rerun the following test to get a more accurate measure?

https://github.com/spotify/featran/commit/352a9f198678ef4ab66bfb01c61154f6bcc2190f
https://github.com/dwyl/english-words/blob/master/words.txt